src/main/java/frc/robot/actions/VisionAlign.java

```diff
@@ -33,10 +32,32 @@
     public double rotationLocalized = 0.0;
     public double navXYawOffset = 0.0;
 
-    
+    public void init()
+    {
+        alignmentStatus = StatusEnum.IN_PROGRESS;
+
+        timeoutCounter = 0.0;
+
+        headingPID = new PID(headingP, headingI, headingD);
+        
+        this.startTime = System.currentTimeMillis();
+    }
 
     public void loop()
     {
+        //TODO: Combine track cancelling into action
```

Comment:
There is a lot of commented code here. The main branch should be only working code. If the code is incomplete, it probably shouldn't be merged yet. If this part is being worked on in another branch it should be merged there first.